docs/releases/1.10.1.txt

```diff
@@ -17,3 +17,6 @@ Bugfixes
 
 * Removed the broken ``BaseCommand.usage()`` method which was for
   ``optparse`` support (:ticket:`27000`).
+
+* Fixed ``migrate`` command to not fail with an ``InconsistentMigrationHistory``
+  exception when there exist unapplied squashed migrations.
```

Comment:
Please add a reference to the ticket using the `:ticket:` directive just like other notes.